test/games/planets_zebeth/layout/test_planets_zebeth_describer.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import dataclasses
+
+import pytest
+
+from randovania.games.game import RandovaniaGame
+from randovania.games.planets_zebeth.layout.planets_zebeth_configuration import (
+    PlanetsZebethArtifactConfig,
+    PlanetsZebethConfiguration,
+)
+from randovania.interface_common.preset_manager import PresetManager
+
+
+@pytest.mark.parametrize(
+    ("artifacts"),
+    [
+        PlanetsZebethArtifactConfig(True, 2),
+        PlanetsZebethArtifactConfig(False, 9),
+        PlanetsZebethArtifactConfig(False, 0),
+    ],
+)
+def test_planets_zebeth_format_params(artifacts):
+    # Setup
+    preset = PresetManager(None).default_preset_for_game(RandovaniaGame.METROID_PLANETS_ZEBETH).get_preset()
+    assert isinstance(preset.configuration, PlanetsZebethConfiguration)
+
+    if not artifacts.vanilla_tourian_keys and artifacts.required_artifacts != 2:
+        # Remove or add missile expansions if Tourian key count is not 2
+        for ammo_def, ammo in preset.configuration.ammo_pickup_configuration.pickups_state.items():
+            if ammo_def.name == "Missile Tank":
+                ammo_ = dataclasses.replace(
+                    ammo,
+                    pickup_count=ammo.pickup_count - (artifacts.required_artifacts - 2),
+                )
+                preset.configuration.ammo_pickup_configuration.pickups_state[ammo_def] = ammo_
+                break
```

Comment:
as said in discord, i'm really not a fan of this condition. either only test artifacts <= 2, or always modify the preset to have ~10 missiles less.

Reply:
can't I use a second preset with 9 keys?

Reply:
If there's a good reason for you to ship a second preset to the end user where the only modification is "9 keys and adjusted items" sure. Otherwise, you can also just use the preset as a test file and load that, like some other tests do.

Reply:
tbf I wanted to provide another starter preset that uses non vanilla keys

Reply:
changed it to use a second preset